src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -37,11 +38,18 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
             this.extensionHost.VisualStateManager.BindingStateChanged += this.SolutionTracker_ActiveSolutionChanged;
             this.solutionTracker.ActiveSolutionChanged += this.SolutionTracker_ActiveSolutionChanged;
             this.CalculateSolutionBinding();
+            this.OnAnalyzersChanged();
```

Comment:
Not really a hack the even handler needs to be setup before the event is occurred and we initialize the package when the solution is opened and has c#/vb projects. 
Since we're in constructor there's no one monitoring for that changed event that you raise here so it should be redundant.